xmlStyle.go

```diff
@@ -80,18 +80,18 @@ type xlsxStyleSheet struct {
 	CellXfs      xlsxCellXfs       `xml:"cellXfs,omitempty"`
 	NumFmts      xlsxNumFmts       `xml:"numFmts,omitempty"`
 
-	theme          *theme
+	theme *theme
+
+	sync.RWMutex   // protects the following
 	styleCache     map[int]*Style
 	numFmtRefTable map[int]xlsxNumFmt
-	lock           *sync.RWMutex
```

Comment:
I'd argue for leaving the `lock` here, because `Alignment` is exported, meaning clients using the library could call methods on the `RWMutex`. Yes, they'd have to go out of their way to do that (or make a "typo"), but better safe than sorry.